src/commands/profile.ts

```diff
@@ -0,0 +1,34 @@
+import { ApplicationCommandOptionType } from 'discord.js';
+
+export const profile: GlobalCommand = {
+	name: 'profile',
+	description: 'Responds with the Profile Picture of the mentioned user',
+	options: [
+		{
+			name: 'user',
+			description: 'The user to get the profile picture of',
+			type: ApplicationCommandOptionType?.User,
+		},
+	],
+	requiresGuild: false,
+	async execute({ interaction, options }) {
+		const user = interaction.user;
+		const param = options[0];
+		if (param?.value === undefined) {
+			await interaction.reply({
+				content: user.avatarURL({ extension: 'png', size: 2048 }) as string,
```

Comment:
Rather than force-casting to `string`, we should probably handle the case where this function returns `null`, perhaps by replying with some error message about the caller not having an avatar.